M2/cmake/build-libraries.cmake

```diff
@@ -1066,6 +1072,38 @@ ExternalProject_Add(build-polymake
   )
 #_ADD_COMPONENT_DEPENDENCY(programs polymake "cddlib;lrs;normaliz;nauty" POLYMAKE)
 
+
```

Comment:
We don't build Bertini, because it is impossible to maintain, due to this clause in the license: "You may modify the Program for your private use only."

Reply:
Sure, which is why the target isn't automatically built. However, providing it is convenient for (my) private use! I read the license, I don't believe adding a build target violates it.

Reply:
Okay: if it's not built by default and does not end up in distributions, so that users come to depend on us providing it to them, then that's fine.

This option would also make it easier to test the Bertini package and to rerun its examples in a github action.

Reply:
Exactly.